### PR TITLE
InstructorFeedbackEditPageUiTest: consolidate similar helper methods #5802

### LINKED ISSUE #5802: InstructorFeedbackEditPageUiTest: consolidate similar helper methods
The test currently has many similar helpers methods like `clickVisibilityOptionsForQuestion1()`, `clickVisibilityOptions(questionNumber)`, and `clickVisibilityOptionsForNewQuestion()`

Actually only the second form is needed.

### PR DESCRIPTION
**Pull request by sujeet14108**
Fixes #5802

### CODE REVIEW

**Comment by chowyb** [comment]
@sujeet14108 is this ready for review?

**Comment by sujeet14108** [comment]
@chowyb Yes, ready for review

**Comment by sujeet14108** [comment]
My bad forgot to push the last commit :(

File: src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java [hunk 1]
```diff
@@ -773,12 +767,14 @@ public void selectConstSumPointsOptionsForNewQuestion(String pointsOption) {
         selectConstSumPointsOptions(pointsOption, NEW_QUESTION_NUM);
     }
 
-    public String getGiverTypeForQuestion1() {
-        return giverDropdownForQuestion1.getAttribute("value");
+    public String getGiverTypeForQuestion(int qnNumber) {
+        WebElement giverDropdownForQuestion= browser.driver.findElement(By.id("questionedittext-" + qnNumber));
```
Comment (whipermr5) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java]: Wrong id; you forgot to change after copy-pasting. Also, add a space before `=`.

File: src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java [hunk 2]
```diff
@@ -773,12 +767,14 @@ public void selectConstSumPointsOptionsForNewQuestion(String pointsOption) {
...
+        return giverDropdownForQuestion.getAttribute("value");
     }
 
-    public String getRecipientTypeForQuestion1() {
-        return recipientDropdownForQuestion1.getAttribute("value");
+    public String getRecipientTypeForQuestion(int qnNumber) {
+        WebElement recipientDropdownForQuestion= browser.driver.findElement(By.id("questionedittext-" + qnNumber));
```
Comment (whipermr5) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java]: Same here.

File: src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java [hunk 3]
```diff
@@ -773,12 +767,14 @@ public void selectConstSumPointsOptionsForNewQuestion(String pointsOption) {
         selectConstSumPointsOptions(pointsOption, NEW_QUESTION_NUM);
     }
 
-    public String getGiverTypeForQuestion1() {
-        return giverDropdownForQuestion1.getAttribute("value");
+    public String getGiverTypeForQuestion(int qnNumber) {
+        WebElement giverDropdownForQuestion = browser.driver.findElement(By.id("questionedittext-" + qnNumber));
```
Comment (whipermr5) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java]: Wrong id.

File: src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java [hunk 4]
```diff
@@ -773,12 +767,14 @@ public void selectConstSumPointsOptionsForNewQuestion(String pointsOption) {
...
+        return giverDropdownForQuestion.getAttribute("value");
     }
 
-    public String getRecipientTypeForQuestion1() {
-        return recipientDropdownForQuestion1.getAttribute("value");
+    public String getRecipientTypeForQuestion(int qnNumber) {
+        WebElement recipientDropdownForQuestion = browser.driver.findElement(By.id("questionedittext-" + qnNumber));
```
Comment (whipermr5) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java]: Same here.

Comment (sujeet14108) [inline src/test/java/teammates/test/pageobjects/InstructorFeedbackEditPage.java]: Done:sweat_smile:

**Pull request merged**